### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by nalch**
Initial version for the package

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+herodotus
```
Comment (gchp) [inline docs/requirements.txt]:
```suggestion
```

File: docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sys
+import os
+import datetime
+
+from herodotus.describe import rpm
```
Comment (gchp) [inline docs/source/conf.py]:
```suggestion
```

File: docs/source/conf.py [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+project = u'version-endpoint'
+copyright = u'2015 - %s, Propylon' % datetime.datetime.now().year
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+
+version = rpm.version()
```
Comment (gchp) [inline docs/source/conf.py]: These need to be updated to use something other than herodotus

Comment (nalch) [reply docs/source/conf.py]: That doesn't seem to be used at all. Removed it and tested the documentation generation.

File: docs/requirements.txt (hunk 1 above)
Comment (nalch) [reply docs/requirements.txt]: Done

**Pull request merged**